c.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/core/src/main/java/org/jruby/ir/IREvalScript.java b/core/src/main/java/org/jruby/ir/IREvalScript.java
```

Comment:
@enebo what is this diff file used for?

Reply:
@edipofederle HAHAHA. thanks I don't know how I ended up committing that file.